deps/libuv.mk

```diff
@@ -24,7 +24,13 @@ else
 UV_FLAGS := --disable-shared $(UV_MFLAGS)
 endif
 
-$(BUILDDIR)/$(LIBUV_SRC_DIR)/build-configured: $(SRCCACHE)/$(LIBUV_SRC_DIR)/source-extracted
+$(SRCCACHE)/$(LIBUV_SRC_DIR)/libuv-unix-signal.patch-applied: $(SRCCACHE)/$(LIBUV_SRC_DIR)/source-extracted
```

Comment:
we do not accept patch files for libuv, please commit the upstream (which we control, and thus can merge easily)

Reply:
Oh, this PR is just a try.
I can re-send the patch to the upstream repo if it works.